src/dbus_api/connection.rs

```diff
@@ -89,6 +93,13 @@ impl DbusConnectionHandler {
                             warn!("Failed to send reply to D-Bus client");
                         }
                     }
+                } else {
+                    let reply = default_reply(&msg);
+                    if let Some(r) = reply {
+                        if connection.send(r).is_err() {
+                            warn!("Failed to send reply to D-Bus client");
+                        }
+                    }
```

Comment:
Can you make this obviously impossible to hang? Right now it still would hang if ```default_reply``` changed its definition, or we for some reason started matching on something other than just a method call.

Reply:
I'm not really sure how we can accomplish that. Are you suggesting simply sending an empty message?

Reply:
We'll let this go for now, since we believe that it solves the problem of hanging completely at this time.